pkg/controllers/csi/livenessprobe/livenessprobe_test.go

```diff
@@ -0,0 +1,158 @@
+package livenessprobe
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+const (
+	driverName = "test-driver-name"
+	healthPort = "9808"
+)
+
+type testCSIServer struct {
+	csi.UnimplementedIdentityServer
+
+	probeTimeout time.Duration
+
+	pluginInfoReqCount int
+	probeReqCount      int
+}
+
+func TestNormalResponse(t *testing.T) {
```

Comment:
why we don't use similar approach with mock like original project https://github.com/kubernetes-csi/livenessprobe/blob/master/cmd/livenessprobe/livenessprobe_test.go#L31 ?